accepted/2020/net5/net5.md

```diff
@@ -493,6 +494,12 @@ Specifically:
   the customer can override this in the project file to a lower version (using a
   higher version than `TargetPlatformVersion` should generate an error).
 
+ * **_TargetFrameworkVersionWithoutV is normally set only internally in the Microsft.NET.Sdk SDK**.
```

Comment:
We do not want to be recomending or encouraging use of internal msbuild properties. There is a better way to do it without them.